docs/minikube.md

```diff
@@ -0,0 +1,74 @@
+## Minikube
+
+Draft requires a wildcard domain and an ingress controller. Here how to set everything
+up when using [Minikube][Minikube]
+
+### Setup the certificate
```

Comment:
I just went along with the existing doc and wanted to correct the assumption that setting a wildcard domain could be hard on minikube. If we don't even need it, even better! I will remove the whole minikube.md since it boils down to making sure that your addon is enabled (the reste was support for the certificate). I will correct the Installation guide with this notice if it is ok.